src/main/resources/org/jenkinsci/plugins/ansible_tower/AnsibleTowerStep/config.jelly

```diff
@@ -33,6 +33,9 @@
     <f:entry field="credential" title="Credential" help="/plugin/ansible-tower/help-credential.html">
         <f:textbox />
     </f:entry>
+    <f:entry field="scmBranch" title="scmBranch" help="/plugin/ansible-tower/help-credential.html">
```

Comment:
This need to be udpated to point to help-scmBranch instead of help-credential.

Reply:
Fixed.